src/modules/transactions/transaction.js

```diff
@@ -49,6 +59,9 @@ class Transaction {
   async cross() {
     throw new Error('You are using a transaction method that requires override')
   }
+  transactionDataHashes() {
+    throw new Error('You are using a transaction method that requires override')
```

Comment:
this looks incorrect, if you add that to later complete it i recommended that the error says something like NOT IMPLEMENTED YET or something like that